packages/keycloak-user-management/src/components/CreateEditUser/index.tsx

```diff
@@ -58,16 +63,35 @@ export const defaultEditUserProps: EditUserProps = {
 
 const CreateEditUser: React.FC<CreateEditPropTypes> = (props: CreateEditPropTypes) => {
   const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [practitioner, setPractitioner] = React.useState<Practitioner | undefined>(undefined);
   const {
     keycloakUser,
     accessToken,
     keycloakBaseURL,
+    opensrpBaseURL,
     serviceClass,
+    opensrpServiceClass,
     fetchKeycloakUsersCreator,
   } = props;
   const userId = props.match.params[ROUTE_PARAM_USER_ID];
   const initialValues = keycloakUser ? keycloakUser : defaultInitialValues;
 
+  const loadPractitioner = async () => {
+    if (userId && practitioner === undefined) {
+      setIsLoading(true);
+      const serve = new opensrpServiceClass(accessToken, opensrpBaseURL, 'practitioner/user');
```

Comment:
Let's move `practitioner/user` to the constants file

Reply:
addressed